headers/csps/chains/mayachain.ts

```diff
@@ -9,7 +9,7 @@ export const csp: Csp = {
   'connect-src': [
     env.VITE_UNCHAINED_MAYACHAIN_HTTP_URL,
     env.VITE_UNCHAINED_MAYACHAIN_WS_URL,
-    env.VITE_MAYACHAIN_NODE_URL,
+    `${env.VITE_MAYACHAIN_NODE_URL}/`,
```

Comment:
eh, why not!

Reply:
just copied from thorchain csp. wasn't working without it with the addition of `/lcd` to the path :shrug: